src/api/routers/config.py

```
@@ -0,0 +1,627 @@
+        # Resolve use_env references
+        base_url = resolve_env_value(request.base_url)
+        api_key = resolve_env_value(request.api_key)
+
+        # Validate required fields
+        if not base_url:
+            return {"success": False, "message": "Base URL is required"}
+        if not request.model:
+            return {"success": False, "message": "Model name is required"}
+
+        # Test by creating client and checking if we can reach the API
+        client = AsyncOpenAI(
```

Ruff (0.14.11)

Local variable `client` is assigned to but never used

Remove assignment to unused variable `client`

(F841)